### PR TITLE
Add faq for containerzed kubelet

### PR DESCRIPTION
**Pull request by cofyc**
for https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/100 and https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/107

### CODE REVIEW

File: docs/faqs.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 - [PV with delete reclaimPolicy is released but not going to be reclaimed](#pv-with-delete-reclaimpolicy-is-released-but-not-going-to-be-reclaimed)
 - [Why my application uses an empty volume when node gets recreated in GCP](#why-my-application-uses-an-empty-volume-when-node-gets-recreated-in-gcp)
 - [Can I change storage class name after some volumes has been provisioned](#can-i-change-storage-class-name-after-some-volumes-has-been-provisioned)
+- [Volume does not exists with containerized kubelet](#volume-does-not-exists-with-containerized-kubelet)
```
Comment (msau42) [inline docs/faqs.md]: typo: exist

File: docs/faqs.md [hunk 2]
```diff
@@ -175,3 +176,20 @@ For now, we don't support migrating volumes to another storage class. If you
...
 storage class, configure discovery directory under new storage class and
 restart all provisioners.
+
+## Volume does not exists with containerized kubelet
+
+If your kubelet is running in a container, it may not be able to access the
+path on the host.
+
+In order to allow the kubelet to access the path on the host, you must prefix
+`hostDir` with the prefix of the host filesystem in the container or mount the
```
Comment (msau42) [inline docs/faqs.md]: clarify that container is "kubelet container"

File: docs/faqs.md [hunk 3]
```diff
@@ -175,3 +176,20 @@ For now, we don't support migrating volumes to another storage class. If you
...
+directory of the local volumes into the container at the same path.
+
+For example, if the root filesystem of the host is mounted at `/rootfs`, you
```
Comment (msau42) [inline docs/faqs.md]: "/rootfs in the kubelet container"

File: docs/faqs.md [hunk 4]
```diff
@@ -175,3 +176,20 @@ For now, we don't support migrating volumes to another storage class. If you
...
+need to prefix the `hostDir` with `/rootfs`. This requires to recreate the local
```
Comment (msau42) [inline docs/faqs.md]: This requires recreating the local PV objects

File: docs/faqs.md [hunk 5]
```diff
@@ -175,3 +176,20 @@ For now, we don't support migrating volumes to another storage class. If you
...
+PV objects. You can delete them all and wait for them to be discovered again.
+
+Another solution is to add a bind in the kubelet deployment configuration to
+mount the parent directory of local volumes into the kubelet container at the
+same path. This requires to restart the kubelet container.
```
Comment (msau42) [inline docs/faqs.md]: This requires restarting

**Comment by cofyc** [comment]
Thanks! Comments addressed.

**Pull request merged**